src/components/overblikk.css

```diff
@@ -0,0 +1,6 @@
+
+
+
+.Panel {
+    margin: 20px;
```

Comment:
Dette blir litt personlig preferanse, men av erfaring synes jeg det er lettere at "parenten" styrer luften mellom "children". Siden `Overblikk`-komponenten lever i konteksten av en parent (i dette tilfellet er det root `div`-en som er parent) så gir det meste mening at `margin` settes her, i stedet for at childen skal gjøre det selv.

En fordel med å la parenten styre dette er at det blir lettere å legge på lik styling på flere children samtidig, i stedet for at man må oppdatere det for hver enkelt child. I tillegg vil det mest trolig være lettere å gjenbruke komponenter i andre kontekster, da man ikke alltid har kontroll på hva andre kontekster/parenter har av layout-styling fra før.

Reply:
Skal endres fra panel til noe annet seinere, bare brukt panel kjapt her.
Skal huske dette til neste gang! :)